### PR TITLE
Provide new field guests on Meeting

### PR DESCRIPTION
**Pull request by Abdu-moustafa**
The PR
- Provided a new field `guests` on the `IWorkspaceMeetingSchema` which allow the users to add `guests` to a meeting.
- Add the guest field to the generated protokoll pdf. 

For [TI-918](https://4teamwork.atlassian.net/browse/TI-918)

## Checklist

- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)

- New translations
  - [x] All msg-strings are unicode

[TI-918]: https://4teamwork.atlassian.net/browse/TI-918?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: opengever/workspace/workspace_meeting.py [hunk 1]
```diff
@@ -88,6 +88,15 @@ class IWorkspaceMeetingSchema(model.Schema):
         default=list()
     )
 
+    form.order_after(guests='attendees')
+    guests = schema.List(
+        title=_(u"label_guests", default=u"Guests"),
+        required=False,
+        value_type=schema.ASCIILine(),
+        default=[],
```
Comment (elioschmutz) [inline opengever/workspace/workspace_meeting.py:96]: beware of mutable defaults:
```suggestion
        default=list(),
        missing_value=list(),
```

Comment (Abdu-moustafa) [reply opengever/workspace/workspace_meeting.py]: Resolved as suggested

File: opengever/workspace/workspace_meeting.py [hunk 2]
```diff
@@ -88,6 +88,15 @@ class IWorkspaceMeetingSchema(model.Schema):
         default=list()
     )
 
+    form.order_after(guests='attendees')
+    guests = schema.List(
+        title=_(u"label_guests", default=u"Guests"),
+        required=False,
+        value_type=schema.ASCIILine(),
```
Comment (lukasgraf) [inline opengever/workspace/workspace_meeting.py:95]: This should be `schema.TextLine()` - email adresses (domains in particular) unfortunately can contain non-ASCII characters.

**Review by lukasgraf — CHANGES_REQUESTED** [review]
LGTM 👍  - just a minor detail plus the thing Elio pointed out with the mutable default values.

File: opengever/workspace/workspace_meeting.py (hunk 2 above)
Comment (Abdu-moustafa) [reply opengever/workspace/workspace_meeting.py]: Changed as suggested

**Review by lukasgraf — APPROVED** [review]
LGTM 👍

**Pull request merged**